string drone_name
string drone_type
---
bool success
uint32 drone_id
string message
